src/main/java/exercise39/SortingRecords.java

```diff
@@ -60,8 +60,11 @@ public String recordSorter(List<Map<String, String>> records) {
     public String recordFilter(String searchTerm, List<Map<String, String>> records) {
 
         String filteredRecords = "";
+        //removeIf?
         for (Map<String, String> map : records) {
+            //can we do some of the "challenges"?
```

Comment:
You seemed to find that pretty straight-forward, should we look at doing some of the _challenges_ in the book?